web/tests/compose_validate.test.js

```diff
@@ -85,6 +85,9 @@ test_ui("validate_stream_message_address_info", ({mock_template}) => {
         user_not_subscribed_rendered = true;
         return html;
     });
+    const $error_banner = $.create("error_banner");
```

Comment:
I'm curious if there is a way to apply DRY (Don't Repeat Yourself) principle when stubbing this. It might resemble the `mock_banners` function, I suppose.

Reply:
I refactored the creation of these banners into `web/tests/lib/compose_banner.js`. Just wanted to note, I'm doing the following:

```
const $error_banner = mock_error_banner();
override_rewire(compose_banner, "parse_single_node", () => $error_banner);
```

instead of the simpler

```
override_rewire(compose_banner, "parse_single_node", () => mock_error_banner());
```

so that the elements get cleared properly after each test. Otherwise I'm getting a "You already created an object with this name" error from zjquery.